### DISCUSSION TITLE
I am not seeing "Manage Access" under "Package Settings"

### DISCUSSION THREAD

**Discussion by alumezi** [community/community#26296] [Actions]
<p><a href="https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility" rel="noopener nofollow ugc">There should be an option for managing access on package settings.</a></p>
<p>But I don’t have it:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/3/83ac5ba41969ff4615b7b5bff5e06a3f0f18e615.png" data-download-href="/uploads/short-url/iMPP50df2qeiuqe3YxxdDDsLDV3.png?dl=1" title="Screen Shot 2022-06-16 at 2.13.59 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/29073778/181096719-c5258a57-06e8-4254-94ac-9853f34bb924.png" alt="Screen Shot 2022-06-16 at 2.13.59 PM" data-base62-sha1="iMPP50df2qeiuqe3YxxdDDsLDV3" width="690" height="171" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83ac5ba41969ff4615b7b5bff5e06a3f0f18e615_2_690x171.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83ac5ba41969ff4615b7b5bff5e06a3f0f18e615_2_1035x256.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83ac5ba41969ff4615b7b5bff5e06a3f0f18e615_2_1380x342.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83ac5ba41969ff4615b7b5bff5e06a3f0f18e615_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-06-16 at 2.13.59 PM</span><span class="informations">3840×954 207 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Akash1134** [comment]
<p>Hi <a class="mention" href="/alumezi">@alumezi</a> ,</p>
<p>Welcome to the GitHub Community and thanks for posting this query. I have asked the engineering team for a review on this. I will get back to you with an update soon.</p>
<p>Thanks!</p>

**Comment by Akash1134** [comment, marked as answer]
<p>GitHub Packages “Manage Access” setting is only available with GHCR right now. Currently, you can only use granular permissions with the Container registry. Granular permissions are not supported in our other package registries, such as the npm registry. For details, see this <a href="https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility">doc</a></p>
